content/en/platform/corda/4.8/open-source/shell.md

```diff
@@ -146,7 +146,13 @@ The standalone shell is a standalone application interacting with a Corda node v
 
 ### Starting the standalone shell
 
-Run the following command from the terminal:
+To run `corda-shell`:
+
+1. Download the shell `.jar` file for your version of Corda—for example, `shell-cli-4.8.jar`.
+
+2. Add the `.jar` file to your environment variables.
```

Comment:
Need to expand on this a bit (I wrote it short in teams).

This is my zshell config for example:

```
alias corda-shell='java -jar "/Users/dannewton/Development/r3/corda-shell/shell-cli/build/libs/shell-cli-4.8-SNAPSHOT.jar"'
```

So maybe something like "Add `corda-shell` to your environment variable which points to the `.jar` file"?